src/components/app/app.js

```diff
@@ -10,68 +10,60 @@ import TaskFilter from "../task-filter/task-filter";
 import AddNewItem from "../add-new-item/add-new-item";
 import SearchTask from "../search-task/search-task";
 
-export default class App extends Component {
-  state = {
-    tasks: [
-      this.createItem("Look at React", true),
-      this.createItem("Start to learning...", true),
-      this.createItem("Make first app", true),
-      this.createItem("Find a job", false),
-      this.createItem("Become a better everyday", false),
-    ],
-    buttons: [
-      { name: "all", label: "All", active: true },
-      { name: "active", label: "Active", active: false },
-      { name: "done", label: "Done", active: false },
-    ],
-    isHidden: false,
-    isActive: "",
-    search: "",
-    firstLoad: true,
-    theme: "dark",
+const App = () => {
+  const [tasks, setTasks] = useState([
+    { label: "Look at React", done: true },
```

Comment:
better create file with mock data and import variables from there 
would keep the code clean

Reply:
Okay, will think about it. Now I have a task delete default values, so it will be fix soon